src/EnrollWebauthnController.js

```diff
@@ -134,32 +134,39 @@ function (Okta, Errors, FormType, FormController, CryptoUtil, webauthn, Footer,
         var children = [];
 
         if (webauthn.isNewApiAvailable()) {
-          //enroll.webauthn.instructions.edge is unescaped because it contains html
+          //enroll.webauthn.v2.instructions.edge is unescaped because it contains html
           children.push(FormType.View({
             View: Okta.View.extend({
               className: 'webauthn-enroll-text',
               template: '\
                 <div class="webauthn-enroll-instructions">\
-                  <p>{{i18n code="enroll.webauthn.instructions" bundle="login"}}</p>\
+                  <p>{{i18n code="enroll.webauthn.v2.instructions" bundle="login"}}</p>\
                 </div>\
                 {{#if isEdge}}\
                   <div class="webauthn-edge-text">\
-                     <p>{{{i18n code="enroll.webauthn.instructions.edge" bundle="login"}}}</p>\
+                     <p>{{{i18n code="enroll.webauthn.v2.instructions.edge" bundle="login"}}}</p>\
+                  </div>\
+                {{/if}}\
+                {{#if onlySupportsSecurityKey}}\
+                  <div class="webauthn-restrictions-text">\
+                     <p>{{{i18n code="enroll.webauthn.instructions.noSupportForBiometric" bundle="login"}}}</p>\
                   </div>\
                 {{/if}}\
                 <div data-se="webauthn-waiting" class="okta-waiting-spinner hide"></div>\
               ',
               getTemplateData: function () {
                 return {
-                  isEdge: BrowserFeatures.isEdge
+                  isEdge: BrowserFeatures.isEdge,
+                  onlySupportsSecurityKey: (BrowserFeatures.isFirefox || BrowserFeatures.isSafari)
+                    && (BrowserFeatures.isMac)
```

Comment:
so if it is safari and macOs, it doesn't support touch id of the macbooks?

Reply:
lol yes!